CHANGELOG.md

```diff
@@ -15,6 +15,9 @@ Increment the:
 
 ## [Unreleased]
 
+* [EXPORTER] Allow to share gRPC clients between OTLP exporters.
```

Comment:
Is this an SDK breaking change? Declare it in the important changes below if yes?

Reply:
It do not break the exported SDK API, but break ABI. Is it still need to declare these changes to declare the important changes?

Reply:
Ok. I think no need then as we don't provide ABI compatibility on SDK.